algorithms/exemplar/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "openie-algorithm-relationshipg3-api",
+  "version": "1.0.0",
+  "description": "Get the Music relationships by running the exemplar Algorithm on a set of sentences.",
+  "main": "index.js",
+  "scripts": {
+	"prestart": "npm install",
+	"start": "node index.js"
+  },
+  "keywords": [
+    "swagger"
+  ],
+  "license": "ISC",
+  "private": true,
+  "dependencies": {
+    "connect": "^3.2.0",
+    "js-yaml": "^3.3.0",
+    "swagger-tools": "0.10.1",
+    "clone": "^2.1.1",
+    "fs": "0.0.1-security"
```

Comment:
Alright, I will create the npm-shrinkwrap.json file and remove the fs module from package.json.